dev/com.ibm.ws.security.audit.source/src/com/ibm/ws/security/audit/encryption/AuditCrypto.java

```diff
@@ -1973,6 +2003,10 @@ static final byte[] encrypt(byte[] data, byte[] key, String cipher, boolean useJ
             } catch (java.security.InvalidAlgorithmParameterException e) {
                 Tr.error(tc, "security.ltpa.noalgorithm", new Object[] { e });
                 com.ibm.ws.ffdc.FFDCFilter.processException(e, "com.ibm.ws.security.audit.AuditCrypto", "2279");
+            } catch (NoSuchProviderException e) {
+                // TODO Auto-generated catch block
+                // Do you need FFDC here? Remember FFDC instrumentation and @FFDCIgnore
+                e.printStackTrace();
```

Comment:
Should this look like the catch above instead of doing `printStackTrace()`?

Reply:
Yes, it should like above too ...